sfdoc/publish/tasks.py

```diff
@@ -103,9 +103,12 @@ def _find_duplicate_urls(url_map):
 
 
 def _scrub_and_analyze_html(
-    html_file, path, article_image_map, images, url_map, problems
+    bundle, html_file, path, article_image_map, images, url_map, problems
 ):
     html = HTML(html_file, path)
+    if html.docset_id and html.docset_id != bundle.docset_id:
```

Comment:
If we are using the `ProductMapUUID` to validate this, should we ever allow it to be blank?

Reply:
No...we should not...

I did get a bit lazy here and commented on it elsewhere.

I have a bunch of finicky zipfiles embedded in zipfiles that I use for integration testing. Getting them all unzipped, fixed and re-zipped correctly was a real headache the last time I did it.

And on the other side we're talking about the output of another automated system. No human inserts these codes. We don't and can't double-check every aspect of Mark's code in SFDoc, so is it worth it for me to do all of that work for the benefit of this one check? I decided not.

Reply:
> Well there is no canonical docset for such a bundle...

Yeah, I was thinking either make something up or just don't give it a name. Throwing an exception when retrieving the index seems likely to accomplish the not naming it part.

Reply:
We could imagine something like 'Super Evil Multiple Docset Bundle'. Maybe that is too much flair.

Reply:
Scripting zip file creation is not horrible... ;)